patches/chrome-browser-ui-content_settings-content_setting_image_model.cc.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/chrome/browser/ui/content_settings/content_setting_image_model.cc b/chrome/browser/ui/content_settings/content_setting_image_model.cc
+index 366ba05a508c8e5f77bd0a380ff229da73065819..b24ef19f81ab2c469780caab71a5953d95750d16 100644
+--- a/chrome/browser/ui/content_settings/content_setting_image_model.cc
++++ b/chrome/browser/ui/content_settings/content_setting_image_model.cc
+@@ -10,6 +10,7 @@
+ #include "base/feature_list.h"
+ #include "base/macros.h"
+ #include "base/metrics/histogram_macros.h"
++#include "brave/browser/ui/content_settings/brave_content_setting_image_models.h"
+ #include "build/build_config.h"
+ #include "chrome/app/vector_icons/vector_icons.h"
+ #include "chrome/browser/browser_process.h"
+@@ -737,6 +738,9 @@ ContentSettingImageModel::GenerateContentSettingImageModels() {
+   std::vector<std::unique_ptr<ContentSettingImageModel>> result;
```

Comment:
couldn't this be done with an override like RegisterWidevineCdmComponent?

Reply:
I don't think so because I'd have to patch the header file to add another definition.